### PR TITLE
Grouped Decompose Helpers for iOS example

### PR DESCRIPTION
**Pull request by ttypic**
Proposal to group Decompose specific helpers in iOS example.

These helpers are quite useful, and when they are in a separate folder, it is easy to find and copy them to your own project.

@arkivanov What do you think?

### CODE REVIEW

**Comment by arkivanov** [comment]
Yeah, I like it! We should also update any links in the docs.

Also, is it common to name folders with spaces?

**Comment by ttypic** [comment]
Great, I am happy you like the idea!

> We should also update any links in the docs.

you are absolutely right, that's why it is in draft now. I will update all docs tomorrow. 

> Also, is it common to name folders with spaces?

I saw it in several projects and official examples, but I guess PascalCase is much more common and will look nicer in urls. I will rename.

File: docs/component/overview.md [hunk 1]
```diff
@@ -217,4 +217,10 @@ struct CounterView: View {
 
 #### What is ObservableValue?
 
-[ObservableValue](https://github.com/arkivanov/Decompose/blob/master/sample/app-ios/app-ios/ObservableValue.swift) is a wrapper around `Value` that makes it compatible with SwiftUI. It is a simple class that conforms to `ObservableObject` protocol. Unfortunately it [does not look possible](https://github.com/arkivanov/Decompose/issues/206) to publish utils for SwiftUI as a library or framework, so it has to be copied to your project.
+[ObservableValue](https://github.com/arkivanov/Decompose/blob/master/sample/app-ios/app-ios/DecomposeHelpers/ObservableValue.swift) is a wrapper around `Value` that makes it compatible with SwiftUI. It is a simple class that conforms to `ObservableObject` protocol. Unfortunately it [does not look possible](https://github.com/arkivanov/Decompose/issues/206) to publish utils for SwiftUI as a library or framework, so it has to be copied to your project.
+
```
Comment (arkivanov) [inline docs/component/overview.md:221]: Let's add a separate section title here, e.g. "#### More Swift utilities".

File: docs/component/overview.md [hunk 2]
```diff
@@ -217,4 +217,10 @@ struct CounterView: View {
...
+* [StackView](https://github.com/arkivanov/Decompose/blob/master/sample/app-ios/app-ios/DecomposeHelpers/StackView.swift) - an adapter for SwiftUI's `NavigationStack` with fallback to `UINavigationView` that works with Decompose's `ChildStack`. See [CountersView.swift](https://github.com/arkivanov/Decompose/blob/master/sample/app-ios/app-ios/DecomposeHelpers/CountersView.swift).
+* [MutableValue](https://github.com/arkivanov/Decompose/blob/master/sample/app-ios/app-ios/DecomposeHelpers/MutableValue.swift) - helps to stub Decompose's `Value`/`MutableValue` for Preview Components. See `PreviewMultiPaneComponent` in [MultiPaneView.swift](https://github.com/arkivanov/Decompose/blob/master/sample/app-ios/app-ios/DecomposeHelpers/MultiPaneView.swift).
+* [SimpleChildStack](https://github.com/arkivanov/Decompose/blob/master/sample/app-ios/app-ios/DecomposeHelpers/SimpleChildStack.swift) - helps to stub Decompose's `ChildStack` for Preview Components. See `PreviewCountersComponent` in [CountersView.swift](https://github.com/arkivanov/Decompose/blob/master/sample/app-ios/app-ios/DecomposeHelpers/CountersView.swift).
```
Comment (arkivanov) [inline docs/component/overview.md]: Incorrect path here? `https://github.com/arkivanov/Decompose/blob/master/sample/app-ios/app-ios/DecomposeHelpers/CountersView.swift`

File: docs/getting-started/quick-start.md [hunk 3]
```diff
@@ -212,7 +212,15 @@ struct RootView: View {
 
 #### What is ObservableValue?
 
-[ObservableValue](https://github.com/arkivanov/Decompose/blob/master/sample/app-ios/app-ios/ObservableValue.swift) is a wrapper around `Value` that makes it compatible with SwiftUI. It is a simple class that conforms to `ObservableObject` protocol. Unfortunately it [does not look possible](https://github.com/arkivanov/Decompose/issues/206) to publish utils for SwiftUI as a library or framework, so it has to be copied to your project.
+[ObservableValue](https://github.com/arkivanov/Decompose/blob/master/sample/app-ios/app-ios/DecomposeHelpers/ObservableValue.swift) is a wrapper around `Value` that makes it compatible with SwiftUI. It is a simple class that conforms to `ObservableObject` protocol. Unfortunately it [does not look possible](https://github.com/arkivanov/Decompose/issues/206) to publish utils for SwiftUI as a library or framework, so it has to be copied to your project. 
+
+You can find more useful utilities for SwiftUI in the [DecomposeHelpers/](https://github.com/arkivanov/Decompose/blob/master/sample/app-ios/app-ios/DecomposeHelpers) folder:
```
Comment (arkivanov) [inline docs/getting-started/quick-start.md]: It looks like a lot of information in quick start docs, I would rather keep only most essential information here. Maybe remove additional things for now? We already have this information in the Component overview page.

**Review by arkivanov — CHANGES_REQUESTED** [review]

**Comment by arkivanov** [comment]
Thank you!

**Pull request merged**